internal/k8shandler/configuration_tmpl.go

```diff
@@ -41,6 +41,7 @@ http.max_header_size: 128kb
 opendistro_security:
   authcz.admin_dn:
   - CN=system.admin,OU=OpenShift,O=Logging
+  - CN=system.admin,OU=Logging,O=OpenShift
```

Comment:
Why does order matter?

Reply:
because SG only does a string comparison... its also not just order that changed